api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -41,6 +41,9 @@ def __init__(self):
     def is_enabled(self) -> bool:
         return self._table is not None
 
+    def get_table_name(self):
+        return self.table_name
```

Comment:
@khvn26 I added this back in. It really simplifies the test [here](https://github.com/Flagsmith/flagsmith/pull/3116/files#diff-637cd4203bacb7503c8d4aa72d88892aa60da8fff0c726d9871ce472f351f062R62) and, in my opinion, we should migrate all other tests to use this approach since the current approach grabs the setting when the module is loaded and then we have to patch the object to change it.